recipes/r-skmeans/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = '0.2-11' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-skmeans
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: skmeans_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/skmeans_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/skmeans/skmeans_{{ version }}.tar.gz
+  sha256: b67aed3ed58399d2b092d00383103a98264db96831706b47f8c05d533de516a9
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-clue >=0.3_39
+    - r-cluster
+    - r-slam >=0.1_31
+
+  run:
+    - r-base
+    - r-clue >=0.3_39
+    - r-cluster
+    - r-slam >=0.1_31
+
+test:
+  commands:
+    - $R -e "library('skmeans')"  # [not win]
+    - "\"%R%\" -e \"library('skmeans')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=skmeans
+  license: GPL-2
+  summary: Algorithms to compute spherical k-means partitions. Features several methods, including
+    a genetic and a fixed-point algorithm and an interface to the CLUTO vcluster program.
+  license_family: GPL2
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-2'  # [win]
+
+extra:
+  recipe-maintainers:
+    - johanneskoester
```

Comment:
Christan is missing here.

Reply:
That's because he isn't listed in the [helper script](https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml). I'll go ahead and add him.

Reply:
Alright, we've increased our bus factor :-)

https://github.com/bgruening/conda_r_skeleton_helper/commit/45ddc80d8130a6aa495be747d78c1608da45c15e